reVX/plexos/rev_reeds_plexos.py

```diff
@@ -4,19 +4,22 @@
 
 @author: gbuster
 """
+import copy
 from concurrent.futures import as_completed
 import json
 import logging
 import numpy as np
 import os
 import pandas as pd
 from scipy.spatial import cKDTree
+from geopandas import GeoDataFrame
 from warnings import warn
 
 from rex.rechunk_h5 import to_records_array
 from rex.utilities.execution import SpawnProcessPool
 from rex.utilities.utilities import parse_table
 
+from reVX.utilities.region import RegionClassifier

```

Comment:
Silly i know but I think we should rename the module region_classifier

Reply:
like the file? yes, agreed!